src/components/journey/errors/JourneyErrorFallback.tsx

```
@@ -0,0 +1,48 @@
+  };
+
+  return (
+    <Card
+      className={cn(
+        'relative',
+        'bg-gradient-to-br from-red-600/95 via-red-700/95 to-red-800/95',
+        'border-none shadow-xl mb-8'
+      )}
+      allowOverflow={true}
+    >
+      <div className="absolute top-0 left-0 w-full h-1 bg-gradient-to-r from-transparent via-red-400/30 to-transparent"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)